services/servers/utils/getLatestMemes.js

```diff
@@ -3,15 +3,28 @@ import {
 }  from '../../../constants/index.js';
 import { getMemeResponseEntity } from './index.js';
 
-export const getLatestMemes = async (client, from, to, size) => {
+export const getLatestMemes = async (client, from, to, size, filtersString) => {
+    const filterObject = filtersString ? JSON.parse(filtersString) : null;
+    const additionalFilter = {};
+    if (filterObject?.channel?.length) {
+        ((additionalFilter.must ??= {}).terms ??= {}).channelName = filterObject.channel.map(channel => channel.toLowerCase());
+    }
+    if (filterObject?.not?.state !== null) {
+        ((additionalFilter.must_not ??= {}).term ??= {}).state = filterObject?.state ?? 1;
```

Comment:
>        ((additionalFilter.must_not ??= {})

You've just set the variable. You know that `additionaFilter.must_not` is `undefined`. Don't check for it, it obscures the intention

Reply:
Same 3 lines above